formulas/formulas.py

```diff
@@ -17,8 +20,15 @@ def quadratic(a, b, c):
     """compute roots for equation of form
     a x^2 + b x + c = 0
     """
+    if a==0:
+        return ["only one root", -c/b]
```

Comment:
good catch, two comments:

1. Lets only return a list of numbers to be consistent
2. Lets keep python2 compatibility, so make sure you are handling integer division properly

Reply:
also, please add a test for this